scala-package/native/src/main/native/ml_dmlc_mxnet_native_c_api.cc

```diff
@@ -1711,3 +1714,435 @@ JNIEXPORT jint JNICALL Java_ml_dmlc_mxnet_LibInfo_mxRtcFree
   int ret = MXRtcFree(handle);
   return ret;
 }
+
+JNIEnv *globalEnv;
+std::unordered_map<std::string, jobject> globalOpPropMap;
+std::unordered_map<std::string, jobject> globalOpMap;
```

Comment:
yes, these object were definded by the user and passed in at runtime, I just define a abstract 
CustomOp and CustomOpProp class. you can see that in the examples.